owslib/feature/wfs100.py

```diff
@@ -260,6 +262,82 @@ def getOperationByName(self, name):
             if item.name == name:
                 return item
         raise KeyError("No operation named %s" % name)
+    
+
+    def get_schema(self, typename, timeout=30):
+        """Parses DescribeFeatureType response and creates schema compatible
+        with :class:`fiona`
+        """
+
+        if not 'DescribeFeatureType' in [op.name for op in self.operations]:
+            return None
+
+        url = self._get_describefeaturetype_url(typename)
+        res = openURL(url, timeout=timeout)
+        tree = etree.fromstring(res.read())
+        typeElementPath = '{%s}element[@name=\'%s\']' % (XS_NAMESPACE, typename)
+        typeElement = tree.find(typeElementPath)
+        complexType = typeElement.attrib['type']
+        elementsPath = '{%(ns)s}complexType[@name=\'%(name)s\']//{%(ns)s}element' %\
+                {'ns': XS_NAMESPACE, 'name': complexType.split(':')[1]}
+        elements = tree.findall(elementsPath)
+        return self._construct_schema(elements)
+
+    def _construct_schema(self, elements):
+
+        schema = {
+            'properties': {},
+            'geometry': ''
```

Comment:
Can we set this to `None` as default? This will make it easier for downstream applications to test more elegantly.